presto-product-tests/conf/docker/common/kerberos.yml

```diff
@@ -1,22 +1,20 @@
 version: '2'
 services:
 
-  # hdp2.5-hive-kerberized contains keytabs and the `krb5.conf` within
-
   hadoop-master:
-    image: 'prestodb/hdp2.5-hive-kerberized:${DOCKER_IMAGES_VERSION}'
+    image: '${HADOOP_MASTER_IMAGE}-kerberized:${DOCKER_IMAGES_VERSION}'
```

Comment:
👍  but please coordinate with https://github.com/prestodb/presto/pull/9550. Or just close this one, as https://github.com/prestodb/presto/pull/9550 contains this change already.